### PR TITLE
Updated mnist_example.py to now work

### PR DESCRIPTION
**Pull request by lukemetz**
Hello,

Sorry to hear the technique didn’t work for you.

I took a look at your code, and your implementation of the unrolling technique is correct. In this pull request I made some hyperparameter changes, and fixed some potential numerical instability. As a result, the model now works.

Sadly this technique does not cure all instability in GANs. They remain very sensitive to hyperparameters. However, we believe unrolling significantly improves stability. (For example, in this case, it enables much simpler training without batchnorm.)

Hope this helps! Thank you for taking the time to reproduce our results!

-Luke

### CODE REVIEW

**Comment by iclrreproducer** [comment]
Fantastic, thanks! The difference between running the updated code with lookahead = 0 and lookahead = 5 is very obvious now.
